### ISSUE TITLE
Updating Collection Types

### ISSUE THREAD

**Issue by The-Outrider** [keystonejs/keystone-demo#32]
I've had to modify my model and currently playing around with the updates functionality. I have a file named 0.0.1-cities.js 

In the file I have the following code:

var keystone = require('keystone');
var Types = keystone.Field.Types;
var Cities = keystone.list('Cities');

module.exports = function(done){
    Cities: [
        {
            photo_file_name: {type: Types.CloudinaryImage, index: true},
        }
    ]
};

When the update runs it hangs in the console no error codes or anything, so while I am pretty sure I am not doing this right, maybe add in some logic to cleanly exit after trying to update for a user defined time?

![screen shot 2016-01-11 at 10 52 35 am](https://cloud.githubusercontent.com/assets/6240904/12241618/73e7bed6-b851-11e5-9b3c-5ea9c0d573d5.png)

node version 4.2.4
npm 2.14.12

Edit // Update: Trying to update the photo_file_name from a String to CloudinaryImage

**Comment by jossmac** [comment]
@The-Outrider it looks like you're combining two different parts of the system.

The `City` list would look something like:
```javascript
var City = new keystone.List('City', {
  ...
});

City.register();
```
More information about [Keystone lists](http://keystonejs.com/docs/database/#lists-example)

The `0.0.1-cities` update script would look something like:
```javascript
var cities = [
  { 'name': 'Sydney', ... },
  ...
];

exports.create = {
  City: cities,
  ...
};
```
More information about [update scripts](https://gist.github.com/JedWatson/10739959)

Let me know how you go :)

**Comment by jossmac** [comment]
Rats, just saw the edit!

This isn't something that update scripts are built for. Do you have Cloudinary urls stored in the string field already?

**Comment by The-Outrider** [comment]
Just realized I'm definitely in the wrong place to submit issues to the main repo ;).

So the update scripts are for the values instead of the fields eh? That's a shame. 

I've logged into my mongo db to swap over the String fields to Object fields, and I've also changed them in the Cities.js model. Ended up manually doing the updates myself after I noticed your comment. 

Thanks for the info :1 :)
